### PR TITLE
Add Falcon Integration Gateway (FIG)

### PR DESCRIPTION
**Pull request by redhatrises**

### CODE REVIEW

File: helm-charts/falcon-integration-gateway/values.yaml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    client_id:
+
+    # Provide OAuth Secret or use FALCON_CLIENT_SECRET in the secrets to override
+    client_secret:
+
+    # Provide application id. Needs to be different per each fig instance.
+    application_id:
+
+    [gcp]
+    # Use GOOGLE_APPLICATION_CREDENTIALS env variable
```
Comment (redhatrises) [inline helm-charts/falcon-integration-gateway/values.yaml:40]: @isimluk not sure what is supposed to go here for the configmap.yaml and values.yaml

File: helm-charts/falcon-integration-gateway/templates/secret.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: {{ include "falcon-integration-gateway.fullname" . }}-creds
+  labels:
+    {{- include "falcon-integration-gateway.labels" . | nindent 4 }}
+data:
+  FALCON_CLIENT_ID: {{ .Values.falcon.client_id | b64enc }}
```
Comment (redhatrises) [inline helm-charts/falcon-integration-gateway/templates/secret.yaml:9]: @isimluk see... base64 encoding provided!

File: helm-charts/falcon-integration-gateway/values.yaml (hunk 1 above)
Comment (isimluk) [reply helm-charts/falcon-integration-gateway/values.yaml:40]: We can remove that section all together.

File: helm-charts/falcon-integration-gateway/values.yaml [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+cloud:
+  google:
+    enabled: true
+    application_credentials:
+
+falcon:
+  client_id:
+  client_secret:
+
+  integration_gateway:
```
Comment (isimluk) [inline helm-charts/falcon-integration-gateway/values.yaml:14]: `s/:$/: |/`

Comment (isimluk) [reply helm-charts/falcon-integration-gateway/values.yaml:14]: Not really. Scratch that.

**Comment by isimluk** [comment]
I think I will close one and re-open in separate PR. I am working on a chart that will take into account all the backends not just google connector.

**Comment by redhatrises** [comment]
Sounds good to me!

**Pull request closed without merging**